### ISSUE TITLE
Have pretrained incepiton-v2 model?

### ISSUE THREAD

**Issue by ysh329** [dmlc/mxnet#5628]
I'm making some benchmarks about inference on some embeded system devices. I only found pretrained inception-v3-batchnorm model on MXNet model zoo. Thus, I wanna ask does anyone have pretrained incepiton-v2-batchnorm model? Thanks a lot! :smile: 
@jspisak

**Comment by tornadomeet** [comment]
http://data.dmlc.ml/mxnet/models/imagenet/inception-bn.tar.gz

**Comment by ysh329** [comment]
@tornadomeet Thanks a lot!
